docs/reference/run.md

```diff
@@ -978,6 +982,16 @@ $ docker run -it \
     --blkio-weight-device "/dev/sda:200" \
     ubuntu
 
+The `--device-read-bps` flag can limit read rate from a device.
+For example, the command creates one container and limits theread rate to `1mb` per second from `/dev/sda`:
+
+    $ docker run -ti --name c1 --device-read-bps /dev/sda:1mb ubuntu:14.04 /bin/bash
+
+The `--device-write-bps` flag can limit write rate to a device.
+For example, the command creates one container and limits write rate to `1mb` per second to `/dev/sda`:
+
+    $ docker run -ti --name c1 --device-write-bps /dev/sda:1mb ubuntu:14.04 /bin/bash
+
```

Comment:
looks like you missed updating this with the changes @moxiegirl gave. Can you update this to
(I also slightly simplified the examples and removed irrelevant options);

```Markdown
The `--read-bps-device` flag limits the read rate from a device. For example,
this command creates a container and limits the read rate to `1mb` per second
from `/dev/sda`:

    $ docker run -it --read-bps-device /dev/sda:1mb ubuntu

The `--write-bps-device` flag limits the write rate to the device. For example,
this command creates a container and limits the write rate to `/dev/sda`: 

    $ docker run -it --write-bps-device /dev/sda:1mb ubuntu

Both flags take limits in the `<device-path>:<limit>[unit]` format. Both read
and write rates must be a positive integer. You can specify the rate in `kb`
(kilobytes), `mb` (megabytes), or `gb` (gigabytes).
```